src/containers/Chat/ChatMessages/ChatMessages.tsx

```
@@ -752,6 +768,19 @@ export const ChatMessages = ({ entityId, collectionId, phoneId }: ChatMessagesPr
   let chatInputSection;
 
   const isSimulatorProp = groups ? false : isSimulator(conversationInfo.contact?.phone);
+  const showCurrentMessages = () => {
+    if (conversationInfo && conversationInfo.messages && conversationInfo.messages[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)